Configurations/CommonBase.xcconfig

```diff
@@ -55,6 +55,13 @@ OTHER_LDFLAGS = $(inherited) $(WK_COMMON_OTHER_LDFLAGS);
 WK_COMMON_OTHER_TAPI_FLAGS = -x objective-c++ -std=c++2a -fno-rtti $(WK_SANITIZER_OTHER_TAPI_FLAGS);
 OTHER_TAPI_FLAGS = $(inherited) $(WK_COMMON_OTHER_TAPI_FLAGS);
 
+WK_DEFAULT_SWIFT_OPTIMIZATION_LEVEL = -O;
+WK_DEFAULT_SWIFT_OPTIMIZATION_LEVEL[config=Debug] = -Onone;
+SWIFT_OPTIMIZATION_LEVEL = $(WK_DEFAULT_SWIFT_OPTIMIZATION_LEVEL);
+
+WK_DEFAULT_SWIFT_VERSION = 5.0;
+SWIFT_VERSION = $(WK_DEFAULT_SWIFT_VERSION);
```

Comment:
I'm a bit surprised that these rules aren't visible to `BaseTarget.xcconfig` or `BaseExtension.xcconfig`. I wonder if we should have a Swift xcconfig file (not necessary for this patch).

Reply:
That is a good point. I think the changes here may not be required. Will fix.

Thanks for reviewing!